### PR TITLE
Shared Tree: update experimental DDS

### PR DESCRIPTION
**Pull request by DLehenbauer**

### CODE REVIEW

File: experimental/dds/tree/api-extractor.json [hunk 1]
```diff
@@ -1,4 +1,15 @@
+/**
```
Comment (DLehenbauer) [inline experimental/dds/tree/api-extractor.json:1]: Added by WB so we stop breaking their doc generation.  (I believe it's harmless.  :-)

File: experimental/dds/tree/README.md [hunk 2]
```diff
@@ -18,15 +18,6 @@ The order of the edits is:
...
 
-## Maintenance
-
-This package is currently being maintained in an internal Microsoft repository as well as in the Fluid Framework repository.
-As a result, there are some inconsistencies in development tooling configuration between this package and a typical Fluid package.
-The main difference is that code is formatted using `prettier`.
-Correct formatting can still be produced using the `lint:fix` script and is enforced with the `lint` script.
-For the most part, this means contributers can ignore these style differences.
-However, this dual maintenance should be kept in mind when considering changes to tooling configuration.
-
```
Comment (DLehenbauer) [inline experimental/dds/tree/README.md:29]: This text is absent in the WB repo, so was removed during sync.

(I think everyone involved has already tripped over this and is aware, so it's probably served it's purpose.)

**Review by SamBroner — APPROVED** [review]
Strong work!

**Comment by DLehenbauer** [comment]
Thanks, @SamBroner !

**Pull request merged**